app/views/reminder_mailings/_form.html.slim

```diff
@@ -0,0 +1,24 @@
+= simple_form_for @reminder_mailing do |f|
+  = simple_error_notice f
+
+  table.table.table-striped
+    thead
+      tr
+        th Auswählen
+        th Freiwilliger
+        th Einsatz
+    tbody
+      = f.simple_fields_for :reminder_mailing_volunteers do |mailing_volunteer|
+        tr
+          td= mailing_volunteer.input_field :selected, as: :boolean
+          td
+            = mailing_volunteer.input_field :volunteer_id, as: :hidden
+            = mailing_volunteer.input_field :reminder_mailable_id, as: :hidden
+            = mailing_volunteer.input_field :reminder_mailable_type, as: :hidden
+            = mailing_volunteer.object.volunteer.to_s
+          td= mailing_volunteer.object.reminder_mailable.to_s
```

Comment:
`.to_s` is optional in views (`=` already calls it for us)

Reply:
Yea, right.
This is not finite yet anyway tho. I'll need to make sure to have a propper to_label method on both GroupAssignment and Assignment, so that I can display this here.

Reply:
Regarding the selection of the volunteers, there is also a "form" here, that as far as I understand either needs to be adapted based on the new changes or removed. 
https://github.com/panter/aoz-003/blob/develop/app/views/reminders/trial_end.html.slim